### DISCUSSION TITLE
Nav. pane "Tree item spacing" : apply it also on file picker

### DISCUSSION THREAD

**Discussion by c266** [Open-Shell/Open-Shell-Menu#1829] [Ideas]
It seems a custom Tree item spacing value is not applied on the navigation pane of the file picker.
Is it possible or outside the scope of Open Shell's possibilities ?

**Comment by Confuciusofessex** [comment]
I've noticed this too. I use -3. There is no change. But open a new tab and it works there. Another But, open a new tab, then that tab will also get the -3 giving a total of -6, open a new tab and now that tab is -9 and after awhile all the tabs will end up at -9. Also the same with a larger spacing (3). I will continue using Q-dir as that has half the (ribbon tool bar what ever it's named these days) height of 2.5cm as explorer has 5cm, on my screen (28" 4k 150%). I also get the slashes instead of the space wasting chevrons (>) and folder size and file count too. Windows 11 23H2 22631.3085
